cmd/manager/main.go

```diff
@@ -144,7 +135,7 @@ func main() {
 			DefaultNamespaces: map[string]crcache.Config{
 				systemNamespace: {LabelSelector: k8slabels.Everything()},
 			},
-			DefaultLabelSelector: dependentSelector,
+			DefaultLabelSelector: k8slabels.Everything(),
```

Comment:
I don't think we want everything. I think we want nothing else. We don't want to accidentally explode our cache with stuff we don't explicitly say we need.

If we just drop this field entirely, is that be semantically equivalent to "nothing else"? I can't remember. It might take a little experimentation.

Reply:
Bryce and I chatted about this more and agreed that an explicit `k8slabels.Nothing()` is the DefaultLabelSelector that we want. That will ensure that nothing makes it into our cache unless we explicitly add an override LabelSelector for it elsewhere in the cache options.